API Homework/UserProfile.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+struct UserProfile: Codable{
+    private let bio : String
+    private let follwersURL: String
+    private let followingURL:String
+    private let avatarURL: String
+    private let public_repos: Int
+    private let location: String
+    private let company: String
+    private let htmlURL: String
+}
```

Comment:
Sao lại để Private trong model vậy.  Để như vậy thì làm sao lây được thuộc tính. Những model để covert json to model nên để  dang optional nhé